e2e/pod.go

```diff
@@ -28,6 +34,25 @@ func getDebugPodName(ctx context.Context, kube *Kubeclient) (string, error) {
 	return podName, nil
 }
 
+// Returns the name of a pod that's a member of the 'debug' deployment, running on an aks-nodepool node.
+func getNonHostDebugPodName(ctx context.Context, kube *Kubeclient, labelName, vmssName string) (string, error) {
```

Comment:
can we rename to something like `findDebugPodNameForVMSS` or something?

`getDebugPodName` above behaves much differently (e.g. doesn't try to search for a particular pod

Reply:
pending rename

Reply:
Coming right up, sorry missed this one